doc/user/content/get-started/quickstart.md

```diff
@@ -43,323 +59,740 @@ Materialize.
+    | auction_house          | load-generator | quickstart |         |
+    | auction_house_progress | progress       | null       |         |
+    | auctions               | subsource      | quickstart |         |
+    | bids                   | subsource      | quickstart |         |
+    | organizations          | subsource      | quickstart |         |
+    | users                  | subsource      | quickstart |         |
     ```
 
-    For now, you'll focus on the `auctions` and `bids` data sets. Data will
-    be **continually produced** as you walk through the quickstart.
+    A [`subsource`](/sql/show-subsources) is how Materialize refers to a table
+    that has the following properties:
 
-1. Before moving on, get a sense for the data you'll be working with:
+    - A subsource can only be written by the source; in this case, the
+      load-generator.
 
-    ```mzsql
-    SELECT * FROM auctions LIMIT 1;
-    ```
-    <p></p>
+    - Users can read from subsources.
 
-    ```nofmt
-     id | seller |        item        |          end_time
-    ----+--------+--------------------+----------------------------
-      1 |   1824 | Best Pizza in Town | 2023-09-10 21:24:54.838+00
-    ```
+1. Use the [`SELECT`](/sql/select) statement to query the subsources/tables. The
+   quickstart uses the `auctions` and `bids` tables.
 
-    ```mzsql
-    SELECT * FROM bids LIMIT 1;
-    ```
-    <p></p>
+    * To view a sample row in `auctions`, run the following
+      [`SELECT`](/sql/select) command:
 
-    ```nofmt
-     id | buyer | auction_id | amount |          bid_time
-    ----+-------+------------+--------+----------------------------
-     10 |  3844 |          1 |     59 | 2023-09-10 21:25:00.465+00
-    ```
+      ```mzsql
+      SELECT * FROM auctions ORDER BY end_time DESC LIMIT 1;
+      ```
+
+      The output should return a single row (your results may differ):
+
+      ```nofmt
+       id    | seller | item               | end_time
+      -------+--------+--------------------+---------------------------
+       29550 | 2468   | Best Pizza in Town | 2024-07-25 18:24:25.805+00
+      ```
+
+    * To view a sample row in `bids`, run the following [`SELECT`](/sql/select)
+      command:
+
+      ```mzsql
+      SELECT * FROM bids ORDER BY bid_time DESC LIMIT 1;
+      ```
+
+      The output should return a single row (your results may differ):
+
+      ```nofmt
+       id     | buyer | auction_id | amount | bid_time
+      --------+-------+------------+--------+---------------------------
+       295641 | 737   | 29564      | 72     | 2024-07-25 18:25:42.911+00
+      ```
+
+
+    * To view the relationship between `auctions` and `bids`, you can join by
+      the auction id. The following query returns the most recently ended
+      auctions and associated bids (limited to 3 rows):
+
+      ```mzsql
+      SELECT a.*, b.*
+      FROM auctions AS a
+      JOIN bids AS b
+        ON a.id = b.auction_id
+      ORDER BY a.end_time desc
```

Comment:
This one ... I think I want it ordered.

Reply:
Actually, will remove the ordering here as well since we're only getting a sample data in these tables.